proposals/per-extension-language-preferences.md

```diff
@@ -0,0 +1,234 @@
+# Proposal: Per-extension language preferences
+
+**Summary**
+
+This proposal allows developers (users) to set a specific language for their extension (which may be different from the browser or system default language) and create a language selection menu for users.
+
+**Document Metadata**
+
+**Author:** hanguokai
+
+**Sponsoring Browser:** Chromium
```

Comment:
I forget if we had already agreed this - was there a discussion somewhere? If not I will take it to the team.

Reply:
@oliverdunk [a meeting note](https://github.com/w3c/webextensions/blob/658a794ac59f014e77a932593950c4188e2ca3a4/_minutes/2024-05-09-wecg.md?plain=1#L135) said this.

Reply:
Cool, let me confirm then. I definitely like the change, I'm just being extra careful any time we add ourselves as the sponsoring browser since we agreed on some strict rules about that meaning we will implement in a timely fashion.

Reply:
Hey Jackie - I spoke to the team about this. We're supportive of the idea, but it is very unlikely we would be able to implement this ourselves any time soon. With that in mind, we'd only be comfortable sponsoring this if there was an external contributor able to implement it.

@xeenon / @Rob--W With that in mind, would either Apple or Mozilla be more likely to implement this soon and be interested in sponsoring?